use-timescale/tigerlake/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: TigerLake
+excerpt: Unifies the Timescale Cloud operational architecture with the data lake (S3 + Iceberg) architectures. This enables real-time application building alongside efficient data pipeline management within a single system.
+products: [cloud]
+keywords: [data lake]
+---
+
+# TigerLake
+
+TigerLake Unifies the Timescale Cloud operational architecture with the data lake (S3 + Iceberg) architectures. This enables real-time application building alongside efficient data pipeline management within a single system.
+
+This experimental release is a native integration enabling continuous replication
+between AWS S3 Tables (managed Iceberg and catalog) running in your AWS account and
+relational tables and hypertables in Timescale Cloud. 
+
+You interact directly with TigerLake using Timescale Console. The pricing associated
+with the integration will be introduced at a later date. This document explains how to
+get started for the early access partners. There are no costs associated with the Timescale Cloud service, we set everything up and then invite you to the project.
+
+## Get started
+
+You first create an S3 table bucket, then use an with an ARN role to enable Timescale Cloud to write to this bucket. Timescale Console supplies a CloudFormation template. You setup TigerCLoud
+using either:
+
+- AWS Management Console
+- AWS cloudformation CLI
+
+
+### Setup TigerLake using AWS Management Console
+
+1. Sign in to the AWS Management Console and open  [CloudFormation console][cmc].
+1. In the navigation bar on the top of the page:
+   1. Choose the name of the currently displayed AWS Region
```

Comment:
I can see that Amazon capitalizes Region, but i don't think we do it anywhere in our docs.